wgpu-types/src/lib.rs

```diff
@@ -4849,14 +4790,34 @@ pub enum LoadOp<V> {
     Clear(V) = 0,
     /// Loads the existing value for this attachment into the render pass.
     Load = 1,
+    /// The render target has undefined contents at the start of the render pass.
+    /// This may lead to undefined behavior if you read from the any of the
+    /// render target pixels without first writing to them.
+    ///
+    /// Blending also becomes undefined behavior if the source
+    /// pixels are undefined.
+    ///
+    /// This is the fastest option on all GPUs if you will always overwrite all pixels
+    /// in the render target.
+    ///
+    /// While you can use this unconditionally, on platforms where `DontCare` is not
+    /// available, it will internally use a different load op.
```

Comment:
found this a bit hard to parse. suggestion
```suggestion
    /// Backends that don't support `DontCare` internally, will pick a different (unspecified) load op instead.
```